### PR TITLE
Fixed Capitalization on data types

### PR DESCRIPTION
**Pull request by ghost**
#### Pre-Submission Checklist
- [x] Your pull request targets the `staging` branch of FreeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/git-rebase#squashing-multiple-commits-into-one) them into one commit).
- [x] All new and existing tests pass the command `npm run test-challenges`. Use `git commit --amend` to amend any fixes.
#### Type of Change
- [x] Bug fix (non-breaking change which fixes an issue)
#### Checklist:
- [x] Tested changes locally.
#### Description

Refer #8392

### CODE REVIEW

**Comment by ltegman** [comment]
I could be wrong but shouldn't `string`, `boolean`, and `number` be uncapitalized as well? My understanding is that the type is lowercase and the object wrapper around the primitive type is uppercase.

**Comment by theflametrooper** [comment]
cc/ @FreeCodeCamp/issue-moderators

**Comment by erictleung** [comment]
I've voiced that I like consistency, either being all lowercase or all uppercase. In the context of this challenge, we are just discussing the datatypes JavaScript has, not the actual object wrapper commands. So I'm for all lowercase. MDN [describes](https://developer.mozilla.org/en-US/docs/Glossary/Primitive) these primitive types all lowercase as well.

@raisedadead this issue gets closer and closer to being bikeshedding 😄

**Comment by raisedadead** [comment]
@erictleung Trust me IMHO, it was good as before the original PR.

And while I agree with the consistency, we just need to consider only one point that matters

**How is a camper (especially new) going to interpret the description?**
@SaintPeter is most correct in his [comment](https://github.com/FreeCodeCamp/FreeCodeCamp/issues/8392#issuecomment-216362953)that some may use `Null` with the `N` capitalized just because they saw it somewhere.

Object wrappers, primitive types these are like huge WORDS for a camper at the stage for this challenge.

TL;DR
Lets keep it simple, for people to understand (with target audience in mind).

**Comment by theflametrooper** [comment]
@raisedadead @erictleung I agree it really is for bike shedding, but we need to set `undefined` and `null` to lower case. I'll update the PR so all items are `lowercase`.

**Comment by ltegman** [comment]
LGTM 👍
